src/ansys/geometry/core/connection/product_instance.py

```diff
@@ -267,11 +272,17 @@ def _is_port_available(port: int, host: str = "localhost") -> bool:
                 return False
 
 
-def _manifest_path_provider(version: int, available_installations: Dict) -> str:
+def _manifest_path_provider(
+    version: int, available_installations: Dict, manifest_path: str = None
+) -> str:
     """Return the ApiServer's addin manifest file path."""
-    return os.path.join(
-        available_installations[version], ADDINS_SUBFOLDER, BACKEND_SUBFOLDER, MANIFEST_FILENAME
-    )
+    if manifest_path != None and os.path.exists(manifest_path):
+        return manifest_path
+    elif manifest_path != None and os.path.exists(manifest_path) == False:
```

Comment:
I'll go one step further and say 😄:
```suggestion
    if manifest_path is not None and os.path.exists(manifest_path):
        return manifest_path
    elif manifest_path is not None and not os.path.exists(manifest_path):
```

Reply:
I can only agree with @jonahrb! =)

Reply:
Done =)